synapse/appservice/__init__.py

```diff
@@ -88,6 +88,8 @@ def __init__(self, token, url=None, namespaces=None, hs_token=None,
         self.sender = sender
         self.namespaces = self._check_namespaces(namespaces)
         self.id = id
+
+        # .protocols is a publicly visible field
```

Comment:
I don't know whether it would be better to add a `get_protocols()` method just to make this more obvious